sensor_bias: [0.8626536285302669, 0.3063923512338875, -0.065159918396569, 0.09316312287734792, -0.2635058187027678, -0.04873327034104846]
mass_matrix: [0.9517813908417049, 0, 0, 0, 0.05680156517438825, -0.003685789167714479, 0, 0.9517813908417049, 0, -0.05680156517438825, 0, 0.000967540349145205, 0, 0, 0.9517813908417049, 0.003685789167714479, -0.000967540349145205, 0]  # matrix to array use matlab reshape, so in columns